examples/coreos/units-etcd/ceph-mon@.service

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=Ceph Monitor
+After=docker.service
+Requires=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+ExecStartPre=-/usr/bin/docker kill %p
+ExecStartPre=-/usr/bin/docker rm %p
+ExecStartPre=/usr/bin/mkdir -p /var/lib/ceph/mon
+ExecStart=/usr/bin/docker run --rm --name %p --net=host \
+   -v /var/lib/ceph:/var/lib/ceph \
+   -e MON_IP=${COREOS_PUBLIC_IPV4}:6789 \
+   -e CEPH_PUBLIC_NETWORK=10.21.147.32/27 \
+   -e CEPH_PRIVATE_NETWORK=172.16.42.0/24 \
+   -e MON_NAME=%H \
+   -e KV_TYPE=etcd \
+   -e KV_IP=127.0.0.1 \
+   ceph/daemon mon
+ExecStopPost=-/usr/bin/docker stop %p
+ExecStopPost=-/usr/bin/docker rm %p
+Restart=always
+RestartSec=10s
+TimeoutStartSec=120
+TimeoutStopSec=15
+
+[X-Fleet]
+Conflicts=ceph-mon@*.service
+
+;; Start on all machines with the mon=true metadata
```

Comment:
I'm pretty sure instance-oriented units can't be used as global units... or at least it will lead to unexpected results.  I think it would be sufficient to document this, rather than issuing separate, non-instance-oriented units (without the `@` suffix).